bin/sozo/src/ops/migration/mod.rs

```diff
@@ -382,6 +362,12 @@ where
         None => {}
     };
 
+    // We don't declare and register the model for Resource Metadata for two reasons:
+    // 1. Having the ModelRegistered event is occupying space for data that are redundant with
+    //    `MetadataUpdate` event.
+    // 2. This model is never used from outside world queries, only to set metadata. It should then
```

Comment:
Hm I think it would be nice to have this automatically indexed by torii though

Reply:
This is what I did initially, but isn't Torii already indexing the metadata update? Do we need both here?

Reply:
Right now we require the registration event to index the model schema first

Reply:
Oh got you. Ok then I can add it back.
As this model is only used by the world internally to store the metadata and serve them when the user requests them, I didn't see the value to have this model indexed.
But you're right, having it registered will ensure the world explorer or other tool based on indexed models can see it. :+1:

Reply:
As for now Torii does not support indexing arrays, as this model contains array, it will not be declared/registered for now. I added a comment to ensure we address that once Torii support is added.